.github/workflows/coverage-test.yml

```diff
@@ -0,0 +1,43 @@
+name: Coverage-test
```

Comment:
It might make more sense to name this file `test-coverage.yml`, instead of `coverage-test`. We also have a convention for workflows, so please name this something like `CI - Test Coverage` if possible.